lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -0,0 +1,121 @@
+package Data::Validate::Sanctions::Fetcher;
+
+use strict;
+use warnings;
+
+use LWP::UserAgent;
+use Text::CSV;
+use XML::Simple qw(:strict);
+use DateTime::Format::Strptime;
+use IO::Uncompress::Unzip qw(unzip $UnzipError);
+use List::Util qw/uniq/;
+use Mojo::UserAgent;
```

Comment:
Why do you need `Mojo::UA` + `LWP::UA`? 

Also, consider sorting the imports alphabetically.